### ISSUE TITLE
Exception on custom AdminConfig 

### ISSUE THREAD

**Issue by dnknth** [nitely/django-app-defaults#1]
Hello @nitely,

I pimped my Django Admin with a custom AdminConfig which is listed in `INSTALLED_APPS` by referencing the class like so:

        'radio.admin.RadioAdminConfig', # custom admin

As per [the docs](https://docs.djangoproject.com/en/3.2/ref/settings/#installed-apps), this option is actually "preferred", even though not many people seem to use it this way.

Unfortunately, this crashes in `django-app-defaults` with the following (abbreviated) stack trace:

```
  File "/Users/dk/src/onair/playout/models.py", line 1, in <module>
    from app_defaults import settings
  File "/Users/dk/src/onair/.venv3/lib/python3.9/site-packages/app_defaults/__init__.py", line 3, in <module>
    from .defaults import (
  File "/Users/dk/src/onair/.venv3/lib/python3.9/site-packages/app_defaults/defaults.py", line 78, in <module>
    settings = Settings(
  File "/Users/dk/src/onair/.venv3/lib/python3.9/site-packages/app_defaults/defaults.py", line 51, in __init__
    if pkgutil.find_loader(module) is None:
  File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/pkgutil.py", line 501, in find_loader
    raise ImportError(msg.format(fullname, type(ex), ex)) from ex
ImportError: Error while finding loader for 'radio.admin.RadioAdminConfig.defaults' (<class 'ModuleNotFoundError'>: No module named 'radio.admin.RadioAdminConfig'; 'radio.admin' is not a package)
```

Any idea how to remedy this?

**Comment by nitely** [comment]
> 'radio.admin' is not a package

is that a package? what's RadioAdminConfig? a python file or a folder?

I think I'd need to know what's inside those folders. If you are on Linux you can use `tree` to print the folder structure.

I've the feeling if you remove the django-app-defaults dependency, it will fail anyway because there is something wrong about that package, but it's bad that is failing in django-app-defaults...

**Comment by dnknth** [comment]
`RadioAdminConfig` lives in `radio/admin.py` and only contains a custom `AdminConfig`.
I include it in `INSTALLED_APPS` instead of `django.contrib.admin`.

```python
from django.contrib import admin
from django.contrib.admin.apps import AdminConfig
from django.utils.translation import ugettext_lazy as _


class RadioAdminConfig(AdminConfig):
    default_site = 'radio.admin.RadioAdminSite'


class RadioAdminSite(admin.AdminSite):
    
    site_header = \
    site_title  = \
    index_title = _('On Air')
```

The package is sound, it is also included in `INSTALLED_APPS` (for templates and static).
The problem at hand is that `django_app_defaults` seems not to be aware that `INSTALLED_APPS`
can also contain names of `AppConfig` classes besides package names.

**Comment by nitely** [comment]
I was not aware modules are valid apps, I thought you can only include
packages...
Well, I fix this later

**Comment by nitely** [comment]
I released v1.2 with the fix
Reopen if it doesn't work

**Comment by dnknth** [comment]
It works now. Thank you for the fast fix!
